lib/jekyll/plugin_manager.rb

```diff
@@ -27,7 +27,7 @@ def require_gems
       site.gems.each do |gem|
         if plugin_allowed?(gem)
           Jekyll.logger.debug("PluginManager:", "Requiring #{gem}")
-          require gem
+          Jekyll::External.require_with_graceful_fail([gem])
         end
```

Comment:
I also notice that require_plugin_files is still using `require` without the graceful_fail method.